plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -140,7 +139,7 @@ func ClusterRoles() []rbac.ClusterRole {
 
 				// additional admin powers
 				rbac.NewRule("create").Groups(authorizationGroup).Resources("localsubjectaccessreviews").RuleOrDie(),
-				rbac.NewRule(ReadWrite...).Groups(rbacGroup).Resources("roles", "rolebindings").RuleOrDie(),
+				rbac.NewRule(ReadWrite...).Groups(rbac.GroupName).Resources("roles", "rolebindings").RuleOrDie(),
```

Comment:
The groups specified in the rules are all string literals.  I think this one should remain string literals too.